santos_akishagelsey/country/countries.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Countries of the World</title>
+    <link rel="stylesheet" href="countries.css">
+</head>
+<body>
+    <h1>Countries of the World</h1>
+    <input type="text" id="countryInput" placeholder="Enter a country">
```

Comment:
ID should be written in snake case. All lower case separated by underscore. 

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwAR2l29lcF09B6N3jV9JjsQn2kIge3yJWLvkiVWepwKW1vg5YT2a6o6bI2Pw#slide=id.g24da8de92d7_0_5